src/test/java/com/meilisearch/sdk/api/documents/DocumentHandlerTest.java

```diff
@@ -169,44 +169,44 @@ void testSearch() throws Exception {
     }
 
     @Test
-    void getUpdate() throws Exception {
+    void getTask() throws Exception {
         when(client.get(any(HttpRequest.class)))
                 .thenAnswer(
                         invocation ->
                                 new BasicHttpResponse(
                                         null,
                                         200,
-                                        "{\"status\":\"processed\",\"updateId\":1,\"type\":{\"name\":\"DocumentsAddition\",\"number\":4},\"duration\":0.076980613,\"enqueuedAt\":\"2019-12-07T21:16:09.623944Z\",\"processedAt\":\"2019-12-07T21:16:09.703509Z\"}"));
-        Update update = classToTest.getUpdate(1);
-        assertNotNull(update);
-        assertEquals("processed", update.getStatus());
-        assertEquals(1, update.getUpdateId());
-        assertEquals("DocumentsAddition", update.getType().getName());
-        assertEquals(4, update.getType().getNumber());
-        assertEquals(0.076980613, update.getDuration());
-        assertEquals("2019-12-07T21:16:09.623944Z", update.getEnqueuedAt());
-        assertEquals("2019-12-07T21:16:09.703509Z", update.getProcessedAt());
+                                        "{\"status\":\"processed\",\"uid\":1,\"type\":{\"name\":\"DocumentsAddition\",\"number\":4},\"duration\":0.076980613,\"enqueuedAt\":\"2019-12-07T21:16:09.623944Z\",\"processedAt\":\"2019-12-07T21:16:09.703509Z\"}"));
+        Task task = classToTest.getTask(1);
+        assertNotNull(task);
+        assertEquals("processed", task.getStatus());
+        assertEquals(1, task.getUid());
+        assertEquals("DocumentsAddition", task.getType().getName());
```

Comment:
These status types have been changed https://github.com/meilisearch/specifications/blob/main/text/0060-tasks-api.md#3-type-field-enum 🍨 

This makes me think a little about the real value of these `DocumentHandler` tests since [we prefer integration tests](https://github.com/meilisearch/integration-guides/blob/main/resources/build-integration.md#5-tests) 🤔

Reply:
You are right, all the code in the `/api` folder is part of the duplicate code we were talking about. I really need to create an issue about it. And yes all these tests are actually mocked I totally forgot to edit them.